content/blog/2022/08/2022-08-05-manage-jenkins-agents-with-podtemplate.adoc

```diff
@@ -0,0 +1,86 @@
+---
+:layout: post
+:title: "Manage Jenkins agents with Kubernetes native PodTemplate"
+:tags:
+- casc
+:author: linuxsuren
+---
+
+Jenkins has great integration with Kubernetes. No matter the controller or agents, they all be able to work well in a Kubernetes Pod. 
+link:https://github.com/jenkinsci/configuration-as-code-plugin[Configuration as Code] is an awesome idea, people don’t need to open the UI page of Jenkins again and again. We can put all the 
```

Comment:
Minor phrasing (and generally preferred to use sentence per line for easier review
```suggestion
link:https://github.com/jenkinsci/configuration-as-code-plugin[Configuration as Code] is an awesome idea.
People don’t need to open the UI page of Jenkins again and again. 
We can put all the 
```